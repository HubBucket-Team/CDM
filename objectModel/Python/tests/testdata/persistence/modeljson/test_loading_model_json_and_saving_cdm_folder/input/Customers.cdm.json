{
  "jsonSchemaSemanticVersion": "0.9.0",
  "imports": [
    {
      "corpusPath": "cdm:/foundations.cdm.json"
    },
    {
      "corpusPath": "custom.extension.cdm.json"
    }
  ],
  "definitions": [
    {
      "entityName": "Customers",
      "exhibitsTraits": [
        {
          "traitReference": "is.extension.pbi:refreshPolicy",
          "arguments": [
            {
              "name": "is.extension.pbi:refreshPolicy",
              "value": {
                "$type": "FullRefreshPolicy",
                "location": "Customers.csv"
              }
            }
          ]
        }
      ],
      "hasAttributes": [
        {
          "name": "CustomerID",
          "isPrimaryKey": false,
          "isReadOnly": false,
          "isNullable": false,
          "dataFormat": "String",
          "sourceOrdering": 0,
          "maximumLength": 0,
          "valueConstrainedToList": false
        },
        {
          "name": "CompanyName",
          "isPrimaryKey": false,
          "isReadOnly": false,
          "isNullable": false,
          "dataFormat": "String",
          "sourceOrdering": 0,
          "maximumLength": 0,
          "valueConstrainedToList": false
        },
        {
          "name": "ContactName",
          "isPrimaryKey": false,
          "isReadOnly": false,
          "isNullable": false,
          "dataFormat": "String",
          "sourceOrdering": 0,
          "maximumLength": 0,
          "valueConstrainedToList": false
        },
        {
          "name": "ContactTitle",
          "isPrimaryKey": false,
          "isReadOnly": false,
          "isNullable": false,
          "dataFormat": "String",
          "sourceOrdering": 0,
          "maximumLength": 0,
          "valueConstrainedToList": false
        },
        {
          "name": "Address",
          "isPrimaryKey": false,
          "isReadOnly": false,
          "isNullable": false,
          "dataFormat": "String",
          "sourceOrdering": 0,
          "maximumLength": 0,
          "valueConstrainedToList": false
        },
        {
          "name": "City",
          "isPrimaryKey": false,
          "isReadOnly": false,
          "isNullable": false,
          "dataFormat": "String",
          "sourceOrdering": 0,
          "maximumLength": 0,
          "valueConstrainedToList": false
        },
        {
          "name": "Region",
          "isPrimaryKey": false,
          "isReadOnly": false,
          "isNullable": false,
          "dataFormat": "String",
          "sourceOrdering": 0,
          "maximumLength": 0,
          "valueConstrainedToList": false
        },
        {
          "name": "PostalCode",
          "isPrimaryKey": false,
          "isReadOnly": false,
          "isNullable": false,
          "dataFormat": "String",
          "sourceOrdering": 0,
          "maximumLength": 0,
          "valueConstrainedToList": false
        },
        {
          "name": "Country",
          "isPrimaryKey": false,
          "isReadOnly": false,
          "isNullable": false,
          "dataFormat": "String",
          "sourceOrdering": 0,
          "maximumLength": 0,
          "valueConstrainedToList": false
        },
        {
          "name": "Phone",
          "isPrimaryKey": false,
          "isReadOnly": false,
          "isNullable": false,
          "dataFormat": "String",
          "sourceOrdering": 0,
          "maximumLength": 0,
          "valueConstrainedToList": false
        },
        {
          "name": "Fax",
          "isPrimaryKey": false,
          "isReadOnly": false,
          "isNullable": false,
          "dataFormat": "String",
          "sourceOrdering": 0,
          "maximumLength": 0,
          "valueConstrainedToList": false
        }
      ]
    }
  ]
}