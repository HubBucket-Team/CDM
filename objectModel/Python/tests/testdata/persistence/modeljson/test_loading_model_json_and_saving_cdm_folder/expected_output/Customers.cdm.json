{
  "imports": [
    {
      "corpusPath": "cdm:/foundations.cdm.json"
    },
    {
      "corpusPath": "custom.extension.cdm.json"
    }
  ],
  "definitions": [
    {
      "entityName": "Customers",
      "exhibitsTraits": [
        {
          "traitReference": "is.extension.pbi:refreshPolicy",
          "arguments": [
            {
              "name": "is.extension.pbi:refreshPolicy",
              "value": {
                "$type": "FullRefreshPolicy",
                "location": "Customers.csv"
              }
            }
          ]
        }
      ],
      "hasAttributes": [
        {
          "dataFormat": "String",
          "name": "CustomerID"
        },
        {
          "dataFormat": "String",
          "name": "CompanyName"
        },
        {
          "dataFormat": "String",
          "name": "ContactName"
        },
        {
          "dataFormat": "String",
          "name": "ContactTitle"
        },
        {
          "dataFormat": "String",
          "name": "Address"
        },
        {
          "dataFormat": "String",
          "name": "City"
        },
        {
          "dataFormat": "String",
          "name": "Region"
        },
        {
          "dataFormat": "String",
          "name": "PostalCode"
        },
        {
          "dataFormat": "String",
          "name": "Country"
        },
        {
          "dataFormat": "String",
          "name": "Phone"
        },
        {
          "dataFormat": "String",
          "name": "Fax"
        }
      ]
    }
  ],
  "jsonSchemaSemanticVersion": "0.9.0"
}